src/files.h

```diff
@@ -0,0 +1,22 @@
+/* Pi-hole: A black hole for Internet advertisements
+*  (c) 2019 Pi-hole, LLC (https://pi-hole.net)
+*  Network-wide ad blocking via your own hardware.
+*
+*  FTL Engine
+*  File prototypes
+*
+*  This file is copyright under the latest version of the EUPL.
+*  Please see LICENSE file for your rights under this license. */
+#ifndef FILE_H
+#define FILE_H
+
+int countlines(const char* fname);
+int countlineswith(const char* str, const char* fname);
+void check_blocking_status(void);
```

Comment:
Should this function be in this header?

Reply:
Yes, the corresponding function is in `files.c`. However, as we read the blocking status from `setupVars.conf`, I will move it to the corresponding module.